docs/blog/tables-for-scientific-publishing/index.qmd

```diff
@@ -0,0 +1,162 @@
+        "cmpd_formula",
+        ps.ends_with("k298")
+    ])
+    .with_columns(
+        cmpd_formula=pl.concat_str(
+            "%" + pl.col("cmpd_formula") + "%"
+        )
+    )
+)
+
+(
+    GT(reactions_mini, rowname_col="cmpd_name")
+    .tab_header(title="Gas-phase reactions of selected mercaptan compounds")
+    .tab_spanner(
+        columns=ps.ends_with("k298"),
+        label="Reaction Rate Constant (298 K),<br>{{cm^3 molecules^–1 s^–1}}"
+    )
+    .fmt_units(columns="cmpd_formula")
+    .fmt_scientific(columns=ps.ends_with("k298"))
+    .sub_missing()
+    .cols_hide(columns="O3_k298")
+    .cols_label(
+        cmpd_formula="",
+        OH_k298="OH",
+        NO3_k298="{{%NO3%}}",
+        Cl_k298="Cl",
+    )
+    .opt_stylize(style=1, color="blue")
+    .opt_horizontal_padding(scale=3)
+    .opt_table_font(stack="humanist")
+)
+```
+
+This is a nice-looking table! And note these pieces:
+
+* The `label=` argument to functions like `.tab_spanner()` supports the use of curly braces (`{{`/`}}`) for the specialized units notation. So using `"{{cm^3 molecules^–1 s^–1}}"` in the input will become cm<sup>3</sup> molecules<sup>–1</sup> s<sup>–1</sup> in the output
+* The `.fmt_units()` method converts values that are already in units notation in the table body. For example, a cell with text `"%CH4S%"` becomes CH<sub>4</sub>S (the surrounding `%` indicates that the text should be interpreted as chemistry notation).
+* The `.fmt_scientific()` method formats values (in this case, very small values) to scientific notation (e.g., 3.50 × 10<sup>–11</sup>). Not doing so would make the table look very strange to a researcher that is familar with this sort of data.
+
+The combination of units notation (and chemistry notation, which is a part of that) really makes the presentation of this table complete and understandable to a practioner of the field. **Great Tables** supports the use of units notation in spanner labels (with `.tab_spanner()`) and also in column labels (with `.cols_labels()`). The column label 'NO<sub>3</sub>' was created with the latter method by supplying the text `"{{%NO3%}}"` as the column label for the `NO3_k298` column.
+
+## Nanoplots
+
+We added the nanoplots feature to **Great Tables** in v0.4.0 (check out the [intro blog post](https://posit-dev.github.io/great-tables/blog/introduction-0.4.0/) for a quick explainer) so that tables can contain small, info-packed plots that fit reasonably well into a table context. They are interactive in that hovering over the data points provides additional plot information. This approach brings together the advantages of plots (elucidation of trends in data) and tables (access to numerical values representing the data points) in a single summary visualization. Version `0.10.0` of **Great Tables** adds the `gibraltar` dataset, which provides meteorological data (temeperature, humidity, wind speed, etc.) for the entire month of May 2024 at Gibraltar Airport Station.
```

Comment:
```suggestion
We added the nanoplots feature to **Great Tables** in v0.4.0 (check out the [intro blog post](https://posit-dev.github.io/great-tables/blog/introduction-0.4.0/) for a quick explainer) so that tables can contain small, info-packed plots that fit reasonably well into a table context. They are interactive in that hovering over the data points provides additional plot information. This approach brings together the advantages of plots (elucidation of trends in data) and tables (access to numerical values representing the data points) in a single summary visualization.

Version `0.10.0` of **Great Tables** adds the `gibraltar` dataset, which provides meteorological data (temeperature, humidity, wind speed, etc.) for the entire month of May 2024 at Gibraltar Airport Station.
```

(So people can scan to the new bit)